pkg/dns/dns.go

```diff
@@ -15,11 +15,14 @@
 package dns
```

Comment:
By reading the commit message, I'm under the impression, it would be good to test such use cases (user blindly hitting Enter key etc.). But it's out of scope I guess.

Reply:
IMO - yes, and yes.